### DISCUSSION TITLE
Proposals for the content of the front-end

### DISCUSSION THREAD

**Discussion by ACSG-64** [ACSG-64/concord-real-time-workspaces#34] [Development]
Propose what content should appear on the front-end.

Suggestions are welcome on a per-page or multi-page basis.

For example:

> **Home page**: this page serves as the landing page of our app. The content could be the following:
> - A banner showing the title of the application, an image and CTA buttons.
> - A section showing the features,
> - ...

**Comment by ACSG-64** [comment]
My proposal for the main page:

- A main section with the title of the application and two CTA buttons, one to 'join now' in the application and the second to 'learn more' about the application.
- A section describing the application briefly.
- A section mentioning the features of the application such as:
  - Each user can create and join multiple workspaces.
  - Real-time communication by chat.
  - The discussion flow could be organized by topics through channels.
- A CTA banner inviting the user to register in the application.

What do you think? feedback is welcome!

**Reply by tkshfj** [reply]
Using [CTA](https://en.wikipedia.org/wiki/Call_to_action_(marketing)) banners sounds fine.
